src/bindgen/parser.rs

```diff
@@ -563,6 +572,71 @@ impl Parse {
         nested_modules
     }
 
+    fn load_syn_impl(&mut self, item_impl: &syn::ItemImpl){
+
+        // Check for trait impl
```

Comment:
```suggestion
```

these comments in the code don't serve much purpose. It's evident from the code what's happening.